src/format.h

```diff
@@ -24,7 +24,7 @@
 
 namespace amber {
 
-/// The format class describes requested image formats. (eg. R8G8B8A8_UINT).
+/// The format class describes requested  data formats. (eg. R8G8B8A8_UINT).
```

Comment:
If this is a format of buffer, we cannot use image formats e.g., `R8G8B8A8_UINT`. I think this example `(eg. R8G8B8A8_UINT)` can be interpreted as "we can use image formats for buffer resource". Please correct me if I misunderstand this.

Reply:
You can define a BUFFER in amberscript with a FORMAT, you then use it as an image buffer.